src/features/keyboard.json

```diff
@@ -0,0 +1,13 @@
+{
+    "findAllReferences": [
+        {
+            "key": "Ctrl-Shift-R", "platform": "mac"
+        },
+        {
+            "key": "Ctrl-Shift-R", "platform": "win"
+        },
+        {
```

Comment:
If the keys are same for all platforms, then just one key should work for all platforms. Refer [this](https://github.com/adobe/brackets/blob/master/src/base-config/keyboard.json).

```
{
     "key": "Ctrl-Shift-R"
}
```